### ISSUE TITLE
Gauge segment boundary can only be integer

### ISSUE THREAD

**Issue by erikgeurts** [selvalt7/modern-circular-gauge#25]
I was trying to define a gauge with a couple of segments, and I wanted one of the segments to start at 4.2 (on a gauge that goes from 0 to 6). However, the UI doesn't accept this value, it will only accept integers.

I tried taking a screenshot, but the moment I do, the mouse-over tool-tip disappears.
![image](https://github.com/user-attachments/assets/4f7e1fb7-050d-4b4e-8cf9-3922b0632bae)

There is a red accent color in the field, and the message in the tool-tip reads:

> Please enter a valid value. The two nearest valid values are 4 and 5.

Perhaps this is by design, perhaps just an oversight. Either way, it would be great to be able to enter a segment boundary with a value other than an integer.

**Comment by selvalt7** [comment]
I forgot that number selector requires to set step size in order to input float numbers.

I think step of 0.1 should be enough, though smaller numbers can be entered in YAML editor.

**Comment by erikgeurts** [comment]
> I think step of 0.1 should be enough, though smaller numbers can be entered in YAML editor.

I agree, that should work for most use cases and users, especially if it can be edited with more precision in YAML.
